docs/components/modeler/bpmn/error-events/error-events.md

```diff
@@ -15,15 +15,14 @@ For example, if an invalid credit card is used in the process below, the process
 In BPMN, **errors** define possible errors that can occur. **Error events** are elements in the process referring to
 defined errors. An error can be referenced by one or more error events.
 
-An error must define an `errorCode` (e.g. `InvalidCreditCard`). The `errorCode` is a `string` used to match a thrown
-error to the error catch events.
+An error must define an `errorCode`. The value of this `errorCode` is used to determine which catch event can catch the thrown error.
 
 For throwing error events, it is possible to define the `errorCode` as an `expression`. When the event is reached,
 the expression is evaluated. An error with the result of this expression is thrown. If no expression is used the
 statically defined `errorCode` is used.
 
-For error catch events, the `errorCode` can be a static value or it can be left empty. An expression can't be used. A
-catch event with an empty `errorCode` will catch **all** thrown errors.
+For catching error events `errorCode` must be a static value.
+Alternatively a catching error event may omit the error reference all together. In this case it catches **all** thrown errors.
```

Comment:
❓ @nikku Why change the naming from error catch event to catching error event?

🔧 I suggest keeping `error catch event` for consistency, AFAIK we typically use the BPMN element type name

Reply:
I've tried to incorporate this into https://github.com/camunda/camunda-platform-docs/pull/2471/commits/0cea47e66891bcc1389575cefd12018a2472c087.